testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewUITests.m

```diff
@@ -35,6 +35,29 @@ - (instancetype)initWithInvocation:(NSInvocation*)invocation {
 }
 
 - (void)testPlatformView {
+  //[self checkGolden];
+  [[XCUIDevice sharedDevice] pressButton:XCUIDeviceButtonHome];
+  [self.application activate];
+  [self checkGolden];
+}
+
+@end
+
+@interface MultiplePlatformViewsBackgroundForegroundTest : GoldenPlatformViewTests
+
+@end
+
+@implementation MultiplePlatformViewsBackgroundForegroundTest
+
+- (instancetype)initWithInvocation:(NSInvocation*)invocation {
+  PlatformViewGoldenTestManager* manager = [[PlatformViewGoldenTestManager alloc]
+      initWithLaunchArg:@"--platform-view-multiple-background-foreground"];
+  return [super initWithManager:manager invocation:invocation];
+}
+
+- (void)testPlatformView {
+  [[XCUIDevice sharedDevice] pressButton:XCUIDeviceButtonHome];
```

Comment:
Suggestion:
It seems like we would encounter a false positive here if Flutter didn't draw the rectangle before going to the background.
We might want to add a golden check before going to the background just to be on the safe side?
Maybe add a new method `checkGoldenWithName` for it?

Reply:
Isn't that covered by the other MultiPlatformViews test (which is drawing a circle on top of the platform views)?